roles/keystone/tasks/main.yml

```diff
@@ -31,6 +31,15 @@
   notify:
     - restart keystone services
 
+- name: install ldap pip reqs in package venv
```

Comment:
I can make a chance in our manifest that adds python-ldap and ldappool to the produced packages.